cardano-api/src/Cardano/Api/IPC.hs

```diff
@@ -480,39 +490,53 @@ queryNodeLocalState :: forall mode result.
                        LocalNodeConnectInfo mode
                     -> Maybe ChainPoint
                     -> QueryInMode mode result
-                    -> IO (Either Net.Query.AcquireFailure result)
-queryNodeLocalState connctInfo mpoint query = do
+                    -> IO (Either QueryError result)
+queryNodeLocalState = queryNodeLocalStateWithVersion NodeToClientV_1
+
+queryNodeLocalStateWithVersion :: forall mode result.
+                       NodeToClientVersion
```

Comment:
Indent?

Also, is there a query that will return available queries for a given node version? Or maybe we can define a fucntion in `cardano-api` like `NodeToClientVersion -> ([QueryInMode mode result], [QueryInEra era])` that  we can use to determine what is available rather than adding an additional function where you have to manually specify the node version.